platforms/allay/src/main/java/org/allaymc/terra/allay/Mapping.java

```diff
@@ -0,0 +1,189 @@
+package org.allaymc.terra.allay;
+
+import com.google.gson.reflect.TypeToken;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+import it.unimi.dsi.fastutil.objects.Object2IntOpenHashMap;
+import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+import lombok.extern.slf4j.Slf4j;
+import org.allaymc.api.block.type.BlockState;
+import org.allaymc.api.block.type.BlockStateSafeGetter;
+import org.allaymc.api.block.type.BlockTypes;
+import org.allaymc.api.item.type.ItemType;
+import org.allaymc.api.item.type.ItemTypeSafeGetter;
+import org.allaymc.api.utils.JSONUtils;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+
+/**
+ * @author daoge_cmd
+ */
+@Slf4j
```

Comment:
If we have to use lombok I'd prefer if we moved to `private static final Logger LOGGER = LoggerFactory.getLogger(Class.class);` for consistency with the rest of the code base